PantryPro.Server/Controllers/GroceryItemType.cs

```diff
@@ -0,0 +1,31 @@
+using Microsoft.AspNetCore.Mvc;
+
+namespace PantryPro.Server.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class GroceryItemTypeController : ControllerBase
+    {
+        private static readonly string[] Descriptions = new[]
+        {
+            "Int", "String", "Array", "Boolean", "Long int", "Char", "Float", "Enum"
```

Comment:
Can you change these to "fruit", "vegetable", "meat", "whole grains", "dairy" and add any other you can think of please.

Reply:
Made the changes, submitting again